server/src/diagnostics.ts

```diff
@@ -191,6 +191,38 @@ export async function perlcritic(textDocument: TextDocument, workspaceFolders: W
     return diagnostics;
 }
 
+export async function perlimports(textDocument: TextDocument, workspaceFolders: WorkspaceFolder[] | null, settings: NavigatorSettings): Promise<Diagnostic[]> {
+    if(!settings.perlimportsLintEnabled) return [];
+    const importsPath = join(getPerlAssetsPath(), 'perlimportsWrapper.pl');
+    let cliParams: string[] = [importsPath].concat(getPerlimportsProfile(settings));
+    cliParams = cliParams.concat(['--lint', '--json', '--filename', Uri.parse(textDocument.uri).fsPath]);
```

Comment:
This part `[importsPath].concat(getPerlimportsProfile(settings))` could be rewritten as `[importsPath, ...getPerlimportsProfile(settings)]` (it's called [spread](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) fyi). I find it nicer to look at than `.concat()`.

Then you could just do a one-liner with `const`. You _should_ be able to drop the type annotation too, TS should figure it out.

```suggestion
    const cliParams = [importsPath, ...getPerlimportsProfile(settings), '--lint', '--json', '--filename', Uri.parse(textDocument.uri).fsPath];
```